metabench.cmake

```diff
@@ -258,8 +231,8 @@ file(WRITE ${MEASURE_RB_PATH}
 # benchmarks. The template is completed by filling it with the contents
 # of the corresponding JSON file and the required JS libraries.
 ################################################################################
-set(CHART_TEMPLATE_HTML_PATH ${CMAKE_CURRENT_BINARY_DIR}/_metabench/chart_template.html.erb)
-file(WRITE ${CHART_TEMPLATE_HTML_PATH}
+set(CHART_HTML_ERB ${CMAKE_CURRENT_BINARY_DIR}/_metabench/chart.html.erb)
```

Comment:
We could also do it the other way around and append `_PATH` to the other variables instead.